### DISCUSSION TITLE
Having problems with installing

### DISCUSSION THREAD

**Discussion by henryernice** [rspber/tsdesktop#5] [Q&A] [answered]
I'm sorry if this is too stupid or basic but i don't really understand what "Resolve links to tsdesktop directories in Arduino/libraries" means. 
I'm not the best programmer so i probably can't help build the community but i hope i can get some help :)

**Comment by rspber** [comment, marked as answer]
Resolving links means configuring directories to point to appropriate places in the file system. This can be done in two ways: either using hard or soft links/junctions or simply copying files from the target directories to the directories listed in the project.
